samples/Messaging/Consumer.Tests/JsonNamingStrategyTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using FluentAssertions;
+using FluentAssertions.Extensions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using PactNet;
+using PactNet.Matchers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Consumer.Tests
+{
+    public class JsonNamingStrategyTests
+    {
+        private readonly JsonSerializerSettings jsonSerializerSettings;
```

Comment:
As mentioned in the MR comment, this can be covered by unit tests instead of integration tests, which is a neater solution. Check out the non-messaging unit tests for pointers.